prow/config.yaml

```diff
@@ -220,6 +220,19 @@ tide:
     istio-ecosystem/authservice: squash
   target_url: https://prow.istio.io/tide
 
+tide:
+  queries:
+  - repos:
```

Comment:
YAML is hard to read, but this is redefining the tide key.

What we want is to add this `queries` list element to the `queries` key on L192, like L193 and L207 both do. (so insert it between L214 and L215).

Also do you want to choose a squash method? If so add it to L216-L217 `istio/operator: squash`

Reply:
right - will address

thanks